resources/sdk/clisdkclient/templates/root.mustache

```diff
@@ -3,16 +3,12 @@ package cmd
 import (
 	"github.com/mypurecloud/platform-client-sdk-cli/build/gc/config"
 	"github.com/mypurecloud/platform-client-sdk-cli/build/gc/logger"
-	"github.com/mypurecloud/platform-client-sdk-cli/build/gc/models"
```

Comment:
Looks like you've removed some necessary stuff from this file.
You can build with `node sdkBuilder.js --sdk clisdkclient` to make sure everything is ok before pushing your changes